### PR TITLE
Cleanup final

### PR DESCRIPTION
**Pull request by buressjan**

### CODE REVIEW

File: src/components/layout/footer/Footer.tsx [hunk 1]
```diff
@@ -5,7 +5,10 @@ function Footer() {
     <FooterWrapper>
       <ContentWrapper>
         <LogoText>
-          Vytvořila <a href='https://sunovy.utb.cz/'>Studentská Unie UTB</a>
+          Vytvořila{' '}
```
Comment (tomesfilip) [inline src/components/layout/footer/Footer.tsx:8]: ayy, could we remove the empty string behind the 'Vytvořila' word? probably a forgotten one

**Review by tomesfilip — CHANGES_REQUESTED** [review]
just a question regarding the prettier, is it going to be implemented later on, or it was just forgotten during this commit? thank you

File: src/components/layout/footer/Footer.tsx (hunk 1 above)
Comment (buressjan) [reply src/components/layout/footer/Footer.tsx:8]: it is there on purpose - it ensures the space between simple text "Vytvořila" and the link "Studentská Unie UTB"

Comment (tomesfilip) [reply src/components/layout/footer/Footer.tsx:8]: yea, cool. also, the &nbsp; could be used for that, but I think it shouldn't be a problem :)

**Review by tomesfilip — APPROVED** [review]
all good, thank you

**Review by tomesfilip — CHANGES_REQUESTED** [review]
Okay sorry, I've missed this.. there's an error while building the app:

src/pages/GetSVG.tsx(1,35): error TS2307: Cannot find module './Reservations.styled' or its corresponding type declarations.

**Review by tomesfilip — APPROVED** [review]

**Pull request merged**